packages/features/ee/workflows/lib/reminders/emailReminderManager.ts

```diff
@@ -42,6 +46,37 @@ async function getBatchId() {
   return batchIdResponse[1].batch_id as string;
 }
 
+function getiCalEventAsString(evt: BookingInfo) {
+  const uid = uuidv4();
+
+  const icsEvent = createEvent({
+    uid,
+    startInputType: "utc",
+    start: dayjs(evt.startTime)
+      .utc()
+      .toArray()
+      .slice(0, 6)
+      .map((v, i) => (i === 1 ? v + 1 : v)) as DateArray,
+    duration: { minutes: dayjs(evt.endTime).diff(dayjs(evt.startTime), "minute") },
+    title: evt.title,
+    description: evt.additionalNotes || "",
+    location: evt.location || "",
+    organizer: { email: evt.organizer.email || "", name: evt.organizer.name },
+    attendees: [
+      {
+        name: preprocessNameFieldDataWithVariant("fullName", evt.attendees[0].name) as string,
+        email: evt.attendees[0].email,
```

Comment:
we would also have to add some more properties like partstat, role, rsvp ..etc   https://github.com/calcom/cal.com/blob/86eb40e0fd79ad54e6c4a01555279e134917041a/packages/emails/templates/attendee-scheduled-email.ts#L61